src/image/root_image.cpp

```diff
@@ -1,49 +1,53 @@
 #include "root_image.hpp"
 
-namespace lucidy{
-   
-    RootImage::RootImage(){
-        settings = settings::IMG::data{1};
-    }
-
-    RootImage::RootImage(const char* path, settings::IMG::data & settings):
-        path(path),
-        settings(settings)
-    {init();}
+namespace lucidy {
 
-     void RootImage::init(){
-        /// <initialize with imread and settings params, Remove temporary code below
-        set(path);
-    }
+RootImage::RootImage() {
+    settings = settings::IMG::data{1};
+}
 
+RootImage::RootImage(const char *path, settings::IMG::data &settings) : path(path), settings(settings) {
+    init();
+}
 
-    void RootImage::set(){
-        if (image.data){ image = cv::Mat(); }
-        cv::String path = getPath();
-        image = cv::imread(path, settings.flag);
-        while (!image.data){
-            std:: cout << "\r\n \t-FILENAME ERROR: Could not find image: " << path << "\r\n";
-            path = getPath();
-            image = cv::imread(path, settings.flag);
-        }
-    }
+void RootImage::init() {
+    /// <initialize with imread and settings params, Remove temporary code below
+    set(path);
+}
 
-    void RootImage::set(const char * new_path){
-        if (image.data){ image = cv::Mat(); }
-        image = cv::imread(new_path, 1);
+void RootImage::set() {
+    if (image.data) {
+        image = cv::Mat();
     }
-
-    cv::Mat RootImage::get(){
-        return image;
+    cv::String path = getPath();
+    image = cv::imread(path, settings.flag);
+    while (!image.data) {
+        std::cout << "\r\n \t-FILENAME ERROR: Could not find image: " << path << "\r\n";
+        path = getPath();
+        image = cv::imread(path, settings.flag);
     }
-
-    cv::String RootImage::getPath(){
-        std::string path;
-        std::cout << "\r\nPlease insert path of RootImage: ";
-        std::cin >> path;
-        std:: cout << "\r\n \t-You inserted: " << path;
-        return cv::String(path);
+}
+
+bool RootImage::set(const char *new_path) {
+    image = cv::imread(new_path, 1);
+    if (!image.data) /// Check for invalid input
```

Comment:
Inconsistent bracket placement, it seems to be common that the bracket should be on the same line as the if statement